build/KoreBuild.sh

```diff
@@ -150,7 +146,7 @@ cat > $msbuildResponseFile <<ENDMSBUILDARGS
 /p:RepositoryRoot="$repoFolder/"
 /fl
 /flp:LogFile="$msbuildLogFile";Verbosity=diagnostic;Encoding=UTF-8
-/p:SakeTargets=$targets
 ENDMSBUILDARGS
+echo -e "$msbuild_args" >> $msbuildResponseFile
```

Comment:
Inlining with the previous statement produced `\n` literals instead of newline characters :-/